File: CursoPOOUber/Java/Car.java
package CursoPOOUber.Java;

public class Car {
    Integer id;        //atributo
    String license;    //atributo
    Account driver;     //atributo
    Integer passenger; //atributo

    public Car(String license, Account driver){
        this.license  = license;
        this.driver   = driver;
    }

    void printDataCar(){
        System.out.println("License: " + license + " Driver: " + driver); //aca creamos una funcion que va a ser llamada como metodo, nos ahorra tener que escribir esta misma funcion y se crear al hacer el objeto a partir de la clase
    }
}

File: CursoPOOUber/Js/car.js
function car(license, driver){
    this.id
    this.license = license
    this.driver = driver
    this.passernger
}

//Esta es la forma antigua

car.prototype.printDataCar = function (){
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
}   